src/cli.js

```diff
@@ -449,6 +450,10 @@ function cli (inputArgs) {
             args.save = true;
         }
 
+        if (args['save-exact']) {
```

Comment:
what happens if you don't have lines 453-456? Are they needed?

Reply:
@stevengill Hmm you are right, I don't think these are needed. I will take it out. The test still passes without these lines and manual tests work too.